### ISSUE TITLE
Vocalization missing using schematics or script intallation

### ISSUE THREAD

**Issue by KateChuen** [SAP/spartacus#14498]
This is the list of things I've noticed are missing when we install spartacus using either the script or schematics. But they work when we install using the develop branch of spartacus repo and using yarn. 

Not sure if it has to do with screen reader. Or there's another issue where some things are just not returned. 

**- My Account - Payment Details** 
- [ ] Set another payment as default - No feedback on "card successfully set as default"

**- Find a store:**
- [ ] when clicking on "View All Stores" - Don't even get Number of Results
- [ ] when clicking on a country from the list of Stores - Don't even get Number of Results
- [ ]
